subsys/bluetooth/controller/hci_driver.c

```diff
@@ -108,6 +108,15 @@ BUILD_ASSERT(!IS_ENABLED(CONFIG_BT_PERIPHERAL) ||
 	#define SDC_PERIODIC_ADV_MEM_SIZE 0
 #endif
 
+#if defined(CONFIG_BT_PER_ADV_SYNC)
+	#define SDC_PERIODIC_ADV_SYNC_COUNT CONFIG_BT_PER_ADV_SYNC_MAX
+	#define SDC_PERIODIC_SYNC_MEM_SIZE \
+		(SDC_MEM_PER_PERIODIC_SYNC(SDC_DEFAULT_PERIODIC_SYNC_BUFFER_COUNT))
```

Comment:
can this be configured by the application somehow? I could imagine if CONFIG_BT_PER_ADV_SYNC_MAX is set to a high value, this could lead to issues. Or if the application does not manage to fetch the events fast enough, e.g. when HCI over SPI is used.

Reply:
I didn't added a config option for this now